Rise-Vision/common-header
# Feature/update volume settings

## Description
I had to update the UI to the latest Style Guide update ( input text is gone, which looks much better )

Related Apps PR: https://github.com/Rise-Vision/rise-vision-apps/pull/1184

## Motivation and Context
UI in style guide was updated here: https://docs.google.com/document/d/1JaurbTMAWsHZIauyrENMVdM1shwdzb5BqtKkjSURVoI/edit#heading=h.2130mvnd6v7c

## How Has This Been Tested?
Validated here: https://apps-stage-9.risevision.com/templates/edit/bf0ffdae-1c9c-4cdc-a847-c68d548043f2/?cid=cf85e5c4-9439-40ce-94d6-e5ea6ea2411c

Validated on mobile also.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
    - Manual testing above
    - To deploy before Friday
    - Simple rollback is enough
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No automated testing needed, no need to notify support.
